### ISSUE TITLE
TT1 Blocks: add styles for the post title

### ISSUE THREAD

**Issue by jffng** [WordPress/theme-experiments#141]
The post title needs styling to match the default theme. It should look like this in the post editor and view: 

<img width="1344" alt="Screen Shot 2020-12-15 at 11 32 57 AM" src="https://user-images.githubusercontent.com/5375500/102243598-5a580a80-3ec9-11eb-88c4-b9b6ea558d5e.png">

**Comment by kjellr** [comment]
Yeah, I'm not sure what to do about this one. https://github.com/WordPress/gutenberg/pull/26355 makes it possible to switch to the full template here and show the post title as it would appear on the front end: 

![template](https://user-images.githubusercontent.com/1202812/102357552-6f41a600-3f7c-11eb-9394-656f519bd6fe.gif)

_(I borrowed this GIF from @jameskoster 🙌)_

That being the case, I'm not totally sure what the plan is for the post title in the editor when you're not editing the full site. Should it borrow the normal `core/post-title` block settings as defined in Global Styles? Or is this meant to be a separate, simpler, "writing" view where the post title wouldn't exactly match the front end?

**Comment by jameskoster** [comment]
> I'm not totally sure what the plan is for the post title in the editor when you're not editing the full site

Yeah this needs some design exploration I think. One idea I've seen floated around is to move the title input to the top bar, kind of like how macOS apps handle renaming:

<img width="469" alt="Screenshot 2020-12-16 at 14 44 49" src="https://user-images.githubusercontent.com/846565/102363470-5ac9d180-3fad-11eb-84c1-13969094e35c.png">

I like this option – at least as a fallback – because it ensures that editing the title is always possible, even if the Title block isn't actually rendered on the canvas. 

It also eliminates any guesswork with regards to where the title will actually appear on the frontend. When the content and template views have different title displays (like the gif above) that can be confusing. The template should always be the source of truth there I think. 

Certainly there are other options, but I haven't seen any other explorations.
